client/app/common/pickupDate/pickupDate.service.js

```diff
@@ -0,0 +1,47 @@
+import base from '../base/service';
+
+class PickupDateComService extends base {
+  constructor($http) {
+    'ngInject';
+    super();
+    this.$http=$http;
+  }
+
+  create(pickup) {
+    return this.$http.post('/api/pickup-dates/', pickup).then(res => res.data);
+  }
+
+  get(params) {
```

Comment:
@D0nPiano this method can also be used to list all pickups. No need for a extra list()/pickupDates() method.
Do you like it? Would introduce this style also to the other services.